common/utils/utils.service.ts

```diff
@@ -0,0 +1,102 @@
+import { encodeAddress, decodeAddress } from '@polkadot/util-crypto';
```

Comment:
I think all of the methods of this class should be static and used like Utils.mothodName istead of this.utils.methodName.